tasks/image_configuration/downgrade-wpa-hostapd/fetch.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+
+ROOT_DST="${DS_OVERLAY}/tmp/"
+
+if [ "${DS_DISTRO}" == "debian" ] ; then
+	LIBSSL_DEB="libssl1.1_1.1.1w-0+deb11u1_armhf.deb"
+	LIBSSL_SITE="http://ftp.us.debian.org/debian/pool/main/o/openssl/${LIBSSL_DEB}"
+	LIBSSL_SHA256="42130c140f972d938d4b4a5ab9638675e6d1223fcff3042bbcc1829e3646eb00"
+
+	WPA_DEB="wpasupplicant_2.9.0-21_armhf.deb"
+	WPA_SITE="http://ftp.us.debian.org/debian/pool/main/w/wpa/${WPA_DEB}"
+	WPA_SHA256="7bb05cb4f6e3fa3a12d10ea900a4bc7d7d606cacc254115ac28583903b55a2a0"
+
+	HOSTAPD_DEB="hostapd_2.9.0-21_armhf.deb"
+	HOSTAPD_SITE="http://ftp.us.debian.org/debian/pool/main/w/wpa/${HOSTAPD_DEB}"
+	HOSTAPD_SHA256="e9bd2195d497e420dfda9a77726c5bc9f79315faee5633517912e06154fe59c2"
+elif [ "${DS_DISTRO}" == "ubuntu" ] ; then
+	LIBSSL_DEB="libssl1.1_1.1.1f-1ubuntu2_armhf.deb"
+	LIBSSL_SITE="http://ports.ubuntu.com/pool/main/o/openssl/${LIBSSL_DEB}"
+	LIBSSL_SHA256="fde1628edbebc3b4aba18f2568b703a4c2003e4903c4e01f899b489f4e426d3f"
+
+	WPA_DEB="wpasupplicant_2.9-1ubuntu4.3_armhf.deb"
+	WPA_SITE="http://ports.ubuntu.com/pool/main/w/wpa/${WPA_DEB}"
+	WPA_SHA256="6d89d49615e903445f0321ddeec6e1d30cbc9fae6fe02cb6aa6f524ae0be969f"
+
+	HOSTAPD_DEB="hostapd_2.9-1ubuntu4_armhf.deb"
+	HOSTAPD_SITE="http://ports.ubuntu.com/pool/universe/w/wpa/${HOSTAPD_DEB}"
+	HOSTAPD_SHA256="b377567609908ec84a573e9c6bb52935ea15874603644cc068904728f19664f1"
+else
+	exit 0
+fi
+
```

Comment:
For the 3 listed sites, can you use:
```
if [ "DS_CUSTOM_APT_MIRROR" == "y" ]; then
    LIBSSL_SITE="$DS_CUSTOM_APT_URL"
    WPA_SITE="$DS_CUSTOM_APT_URL"
    HOSTAPD_SITE="$DS_CUSTOM_APT_URL"
 fi

Reply:
I would push back on that for a couple of reasons.

1) It's not an APT URL, its a direct link to the relevant `deb` package and are not in APT URL format. They are not downloaded through any of the APT tools. Unless there is a way to do this that I am missing?
2) According to help, DS_CUSTOM_APT_URL is for the initial fetch. Which, if I set it up to use the older bullseye, that would conflict with the bookworm default repo, no?

Reply:
TLDR, you're right, and we should not use the custom apt mirror.

The APT_URL is what is currently what is used to build the multistrap.conf. Looking closer, you're right that they aren't the same format:
https://github.com/embeddedTS/distro-seed/blob/main/tasks/core/create_multistrap_conf/create_multistrap_conf.sh#L13

1) My biggest concern is that we dont hammer debian's repos when our CI is set up. That is the main point of the CUSTOM_APT_MIRROR so we can use our local mirror. Since this uses fetch_blob that should cache and this should be harmless anyway.  Thinking this through, our mirror might not have those specific versions anyway.

2) This is likely moot, but otherwise using this initial URL would be fine for using as a download link.